plugins/justify/plugin.js

```diff
@@ -178,8 +188,18 @@
 
 		refresh: function( editor, path ) {
 			var firstBlock = path.block || path.blockLimit;
+			var name = firstBlock.getName();
+
+			// Check if we can give to element proper class, style or current path is body (#455).
+			// IE8, after focus and refresh return path with 'body'
+			if ( editor.config.enterMode === CKEDITOR.ENTER_BR && firstBlock === editor.editable() ) {
```

Comment:
This condition will probably never be true. Mind the fact that path will contain _new_ instances of `CKEDITOR.dom.element` for every fetched element. Therefore there is very little chance that `editor.editable` would return one of them. To check the equality of two nodes, use [`CKEDITOR.dom.element.prototype.equals`](http://docs.ckeditor.com/#!/api/CKEDITOR.dom.domObject-method-equals).

More over this change introduces the strange behavior in IE8 in `justifyrespectacf` test. When editable is selected, justify buttons are disabled. After clicking inside paragraph, the buttons are still disabled and I can't change the alignment of the paragraph. Honestly I liked the previous version more – there was only the issue with checking name of the block, not the block itself.